tinymce-angular-component/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@tinymce/tinymce-angular",
   "description": "Official TinyMCE Angular Component",
-  "version": "3.5.0",
+  "version": "",
```

Comment:
intended?

Reply:
Yep, it's now set to match the main package.json during the build process.